src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi

```diff
@@ -203,6 +203,13 @@ cdef class _ServicerContext:
     def invocation_metadata(self):
         return self._rpc_state.invocation_metadata()
 
+    def is_active(self):
+        return self._rpc_state.status_code == StatusCode.ok
```

Comment:
This is probably wrong, we might want to check if it is `None` instead. If a status code is given, the RPC is finished.